core/utils/samples.py

```diff
@@ -490,15 +490,10 @@ def get_sample_obj_from_sample_name(sample_name):
     return None
 
 
-def get_sample_obj_from_fingerprint(sample_fingerprint):
-    """Return the sample instance from its unique fingerprint"""
-    if core.models.Sample.objects.filter(
-        sample_fingerprint__iexact=sample_fingerprint
-    ).exists():
-        return core.models.Sample.objects.filter(
-            sample_fingerprint__iexact=sample_fingerprint
-        ).last()
-    return None
+def get_sample_obj_from_unique_sample_id(unique_sample_id):
```

Comment:
have you checked this function is not used anywhere else? If so maybe it's better to create another function not modify this one